personio-auth-api.yaml

```diff
@@ -17,17 +17,20 @@ paths:
       description: 'Request Authentication Token'
       produces:
         - application/json
-      parameters:
-        - name: client_id
-          in: query
-          required: true
-          type: string
-          description: Client id of the downloaded credentials file
-        - name: client_secret
-          in: query
-          required: true
-          type: string
-          description: Client secret of the downloaded credentials file
+      requestBody:
+        content:
+          application/json:
```

Comment:
or should this be multipart form?

Reply:
in practice, both work..

Reply:
I personally would go for the `requestBody` approach that you already have here

Reply:
actually, i can put both down, and then it should be rendered as "either or"